scripts/get_all_heron_paths.sh

```diff
@@ -21,8 +21,8 @@ set -eu
 set +e
 # Build everything
 DIR=`dirname $0`
-source ${DIR}/detect_os_type.sh
```

Comment:
This script is used in the files below as well, can we drop/migrate usage there as well? If so it would eliminate the usage of the script and we could delete it.

https://github.com/apache/incubator-heron/blob/b12a3965f4acf1ac3f26c27ae4ead624afe706ce/scripts/run_integration_test.sh#L78

https://github.com/apache/incubator-heron/blob/b12a3965f4acf1ac3f26c27ae4ead624afe706ce/scripts/run_integration_topology_test.sh#L40

Reply:
Yes I think you are correct. I missed the other references. Will remove and do another test.